airflow/executors/debug_executor.py

```diff
@@ -118,6 +119,11 @@ def trigger_tasks(self, open_slots: int) -> None:
 
         :param open_slots: Number of open slots
         """
+        if not self.queued_tasks:
+            # wait a bit if there is no task ready to be executed to avoid spinning too fast in the void
```

Comment:
```suggestion
            # wait a bit if there are no tasks ready to be executed to avoid spinning too fast in the void
```

Reply:
never too sure with those, I was thinking zero is singular :p
thanks for fixing !